File: python/rna-transcription/rna_transcription.py
def to_rna(dna_strand):
    dna_list = list(dna_strand)

    for letter in range(len(dna_strand)):

        if(dna_list[letter] == 'G'):
            dna_list[letter] = 'C'

        elif(dna_list[letter] == 'C'):
            dna_list[letter] = 'G'

        elif (dna_list[letter] == 'T'):
            dna_list[letter] = 'A'

        elif (dna_list[letter] == 'A'):
            dna_list[letter] = 'U'
    return ''.join(dna_list)

File: javascript/rna-transcription/rna-transcription.js
//
// This is only a SKELETON file for the 'RNA Transcription' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const DNAToRNA = {
    G: "C",
    C: "G",
    T: "A",
    A: "U",
};

export const toRna = (str) => {
    let result = "";
    if (str) {
        for (const letter of str) {
            result += DNAToRNA[letter];
        }
    }
    return result;
};
